src/Components/Steps/Steps.module.css

```diff
@@ -0,0 +1,149 @@
+.stepsContainer {
+  display: flex;
+  flex-direction: column;
+}
+
+.steppingContainer {
+  display: flex;
+  flex-direction: row;
+  cursor: pointer;
+  /* align-items: flex-start; */
+  /* justify-content: flex-start; */
+}
+.disbContainer {
+  position: relative;
+  margin: 10px -10px;
+  height: 45px;
+}
+
+.disbSteppingContainer {
+  position: absolute;
+  height: 50px;
+  left: 0;
+  background-color: var(--disabled-grey-background);
+  display: flex;
+  width: 200%;
+  border-radius: 5px;
+  font-weight: 500;
+  padding: 10px 0;
+}
+.stepNumber {
+  background-color: var(--purple);
+  width: 25px;
+  height: 25px;
+  margin-right: 7px;
+  text-align: center;
+  color: var(--white);
+  border-radius: 50%;
+  display: inline-block;
+}
+.stepText {
+  color: var(--black);
+  display: inline-block;
+}
+.disbStepNumber {
```

Comment:
`stepNumber` and `disbStepNumber` share a lot of properties. Recommend using base and modifier classes here as described in more detail below. Let me know if you're not sure what I mean or how to achieve this.